src/sync/incomingSyncUtils.js

```diff
@@ -504,7 +504,7 @@ export const createOrUpdateRecord = (database, settings, recordType, record) =>
       try {
         const parsedData = JSON.parse(data);
         const shouldSetData = checkIsObject(parsedData);
-        internalRecord = { id, title, type, _data: shouldSetData ? data : null };
+        internalRecord = { id, title, type, _data: shouldSetData ? data.data : null };
```

Comment:
Lets just do this one here too if you don't mind: https://github.com/openmsupply/mobile/issues/1878

If you see here: https://github.com/sussol/msupply/pull/4985 - actually it is not called `data`! it is `json` .. so above we need to change `data` to `json` and also it should be `json.data`.

also here: https://github.com/openmsupply/mobile/blob/8490cbec79a165ea9e0d362c69b0b3a8ca448d67/src/sync/incomingSyncUtils.js#L255

Reply:
Great that you noticed that! 
Sure, I'' work on that now. I didn't see #1878 because it wasn't in the Project Dashboard (Found you're a human been after all haha)

Reply:
🤖

Reply:
> Found you're a human been after all haha
no way 🤯

Reply:
Done here 6ac1fc6

Reply:
@joshxg 
I guess this one `_data` should be `_json` as well, don't you think? I missed that

Reply:
I personally hate the name `json` for a field 🤣  So I think `data` is fine. What do you think?

Reply:
Yeah haha let's keep that `_data`, at least as much as we can